plugins/reach/lib/api/enums/KalturaCatalogItemLanguage.php

```diff
@@ -97,4 +97,7 @@ class KalturaCatalogItemLanguage extends KalturaStringEnum
 	const ZH_CN = KalturaLanguage::ZH_CN;
 	const ZH_HK = KalturaLanguage::ZH_HK;
 	const AUTO_DETECT = "Auto Detect";
+
+	const EN_ASL = "English (ASL)";
```

Comment:
@hilak Will the client not agree to use the existing languages ​​(EN_US/EN_GB)? Because its basically a modification of the enum method that uses KalturaLanguage languages ​​and not specific strings.